docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -261,15 +261,13 @@ Sentry will only comment on pull requests less than two weeks old that are assoc
 
 #### Open Pull Request Comments
 
-<Include name="feature-available-for-user-group-early-adopter.mdx" />
-
 When a pull request is opened, Sentry will parse the filenames and function names modified in the pull request from the PR diff to find recent unhandled, unresolved issues associated with your pull request.
 
 ![Sentry comment on open pull request in GitHub](/product/integrations/source-code-mgmt/github/sentry-github-open-pr-comment.png)
 
 Sentry will comment on the pull request with up to five issues per file that were first seen within the past 90 days and were last seen within the past 14 days.
 
-This feature requires [code mappings](/product/issues/suspect-commits/#2-set-up-code-mappings) and is currently only supported for Python files.
+This feature requires [code mappings](/product/issues/suspect-commits/#2-set-up-code-mappings) and is currently only supported for Python and Javascript/Typescript files. For Javascript/Typescript, please ensure that you've unminified your code in Sentry.
```

Comment:
noob question 😅 do everyone know how to unminify code in Sentry? If not I think we need a link to a support doc here for that.

Reply:
there's usually a button in the stacktrace... let me try to find an example

Reply:
Don't worry too much. It might be just something that I'm not familiar with.

Reply:
actually the correct phrasing should be "make sure to configure source maps for unminified code" or something
<img width="610" alt="Screenshot 2024-02-09 at 11 59 44" src="https://github.com/getsentry/sentry-docs/assets/70817427/1dc221e9-b35a-44fb-adbd-ea5e81cc3281">